### PR TITLE
Added changes related to Drupal 9 compatibility

### PR DESCRIPTION
**Pull request by andriyun**
During the work on the upgrade, https://github.com/OS2Forms/OS2Forms8 to Drupal9 was there a need to update some modules to their higher version.

This PR includes changes related to Drupal 9 compatibility

The effect of these changes can be tested on OS2Forms8 solution by using code from d9 brunch https://github.com/OS2Forms/os2forms8/tree/d9

It's added to OS2Forms8 via replacing of version os2forms_forloeb_profile on the line https://github.com/OS2Forms/os2forms8/blob/d9/composer.json#L118

It is supposed to be changed to a new release tag afterward.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "drupal/gin": "^3.0",
-        "drupal/core": "^8",
-        "os2forms/os2forms": "^2.6",
+        "drupal/core": "^8 || ^9",
+        "os2forms/os2forms": "^3.0",
         "os2forms/os2forms_forloeb": "^1.7",
```
Comment (rimi-itk) [inline composer.json:22]: I think we should upgrade `os2forms/os2forms_forloeb` to `^1.8`, i.e. the latest version.

**Review by rimi-itk — COMMENTED** [review]
See comment on `os2forms/os2forms_forloeb` version.

**Comment by madsnorgaard** [comment]
We should merge to `develop` before `main`.

**Comment by andriyun** [comment]
Hi @madsnorgaard 
I've changed the destination branch for PR from `main` to `develop`.

After this change, I see that PR got more commits to merge + merge conflict in `composer.json` on top of it.
It looks like `develop` branch has not gotten all commits that the `main` branch has.
See more here https://github.com/OS2Forms/os2forms_forloeb_profile/compare/develop...main

Is it something you can manage to solve on your side?

**Comment by madsnorgaard** [comment]
I believe this is ready to merge.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**